config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -2393,28 +2389,25 @@ presubmits:
     - release-1.13
     cluster: security
     context: pull-security-kubernetes-typecheck
+    decorate: true
     labels:
       preset-service-account: "true"
     name: pull-security-kubernetes-typecheck
+    path_alias: k8s.io/kubernetes
     rerun_command: /test pull-security-kubernetes-typecheck
     spec:
       containers:
       - args:
         - --ssh=/etc/ssh-security/ssh-security
-        - --job=$(JOB_NAME)
-        - --root=/go/src
-        - --repo=github.com/kubernetes-security/$(REPO_NAME)=$(PULL_REFS)
-        - --service-account=/etc/service-account/service-account.json
-        - --upload=gs://kubernetes-security-prow/pr-logs
-        - --timeout=75
-        - --scenario=execute
-        - --
         - make
         - verify
-        - WHAT=typecheck
-        - KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/_artifacts
+        command:
+        - runner.sh
```

Comment:
What does the runner do?

Reply:
runner gives some hacky things like... GOPATH in PATH, activate service account, enable bazel cache, docker-in-docker, etc, wonder if you need any of those here though :-)

Reply:
Why do we need the service account? Bootstrap needs this to do the gcs upload, but this is now handled by other containers

Reply:
I'm not exactly sure what the job itself is doing, if it doesn't do things like gcs calls you can get rid of the service account preset as well.

Reply:
We should do `$GOPATH/bin` in `$PATH` in the decoration